### PR TITLE
Remove Outbrain

### PR DESCRIPTION
**Pull request by shtukas**
## What does this change?

Remove support for Outbrain. 

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No. (Already done)

### CODE REVIEW

**Comment by shtukas** [comment]
Plista working fine on CODE in Australia:

x

**Comment by shtukas** [comment]
Bon débarras !

File: common/app/views/fragments/footer.scala.html [hunk 1]
```diff
@@ -146,7 +146,7 @@
             </a>
             <div class="really-serious-copyright">
                 @if(!page.metadata.isFoundation) {
-                    © @{new DateTime().year.getAsText} Guardian News &amp; Media Limited or its affiliated companies. All rights reserved.
+                    © @{new DateTime().year.getAsText} Guardian News &amp; Media Limited or its affiliated companies. All rights reserved [16].
```
Comment (sndrs) [inline common/app/views/fragments/footer.scala.html:149]:
```suggestion
                    © @{new DateTime().year.getAsText} Guardian News &amp; Media Limited or its affiliated companies. All rights reserved.
```

**Review by sndrs — COMMENTED** [review]
should `plista-outbrain-renderer.js` etc be renamed e.g. `plista-renderer.js`?

**Comment by buck06191** [comment]
> should `plista-outbrain-renderer.js` etc be renamed e.g. `plista-renderer.js`?

Agree on this. Makes it clearer that outbrain is gone.

File: common/app/conf/switches/FeatureSwitches.scala [hunk 2]
```diff
@@ -57,16 +57,6 @@ trait FeatureSwitches {
...
-    SwitchGroup.Feature,
-    "outbrain",
-    "Enable the Outbrain content recommendation widget on web and AMP.",
-    owners = Seq(Owner.withGithub("johnduffell")),
-    safeState = Off,
-    sellByDate = never,
-    exposeClientSide = true
-  )
-
   val PlistaForOutbrainAU = Switch(
```
Comment (buck06191) [inline common/app/conf/switches/FeatureSwitches.scala:60]: Can we rename this to remove the Outbrain reference?

File: static/src/javascripts/projects/commercial/modules/third-party-tags/plista-outbrain-renderer.spec.js [hunk 3]
```diff
@@ -47,7 +40,6 @@ describe('Plista Outbrain renderer', () => {
             config.set('switches.plistaForOutbrainAu', true);
             config.set('page.edition', edition);
             initPlistaOutbrainRenderer().then(() => {
```
Comment (buck06191) [inline static/src/javascripts/projects/commercial/modules/third-party-tags/plista-outbrain-renderer.spec.js:42]: Can we rename this to `initPlistaRenderer()`?

**Review by buck06191 — CHANGES_REQUESTED** [review]
Looks generally good to me with a few renaming options

File: common/app/views/fragments/footer.scala.html (hunk 1 above)
Comment (shtukas) [reply common/app/views/fragments/footer.scala.html:149]: Yeah, it was part of the review (I promised I would remove it before merging ^^)

File: common/app/conf/switches/FeatureSwitches.scala (hunk 2 above)
Comment (shtukas) [reply common/app/conf/switches/FeatureSwitches.scala:60]: Done

File: static/src/javascripts/projects/commercial/modules/third-party-tags/plista-outbrain-renderer.spec.js (hunk 3 above)
Comment (shtukas) [reply static/src/javascripts/projects/commercial/modules/third-party-tags/plista-outbrain-renderer.spec.js:42]: Done

**Review by buck06191 — APPROVED** [review]
LGTM

File: static/src/javascripts/projects/commercial/modules/check-dispatcher.js [hunk 4]
```diff
@@ -68,39 +56,6 @@ const checksToDispatch = {
...
-
-        return Promise.all(dependentChecks).then(results =>
-            everyCheckPassed(results)
-        );
-    },
-
-    isOutbrainMerchandiseCompliant(): Promise<boolean> {
-        const dependentChecks = [
-            waitForCheck('hasHighPriorityAdLoaded'),
-            waitForCheck('hasLowPriorityAdNotLoaded'),
```
Comment (mchv) [inline static/src/javascripts/projects/commercial/modules/check-dispatcher.js:86]: @shtukas I think the 2 function `hasHighPriorityAdLoaded` can `hasLowPriorityAdNotLoaded` can as well be removed as I think  they are used only by `IsOutbrainMerchandiseCompliant`.

Comment (shtukas) [reply static/src/javascripts/projects/commercial/modules/check-dispatcher.js:86]: Done!

File: static/src/javascripts/projects/commercial/modules/dfp/define-slot.js [hunk 5]
```diff
@@ -111,7 +97,7 @@ const defineSlot = (adSlotNode: Element, sizes: Object): Object => {
         if (isEligibleForOutstream(slotTarget)) {
             allowSafeFrameToExpand(slot);
         }
-        slotReady = setHighMerchSlotTargeting(slot, slotTarget);
+        slotReady = Promise.resolve();
```
Comment (ioanna0) [inline static/src/javascripts/projects/commercial/modules/dfp/define-slot.js:100]: Amazing! Just having a quick look I notice `slotReady` is now always set to `Promise.resolve()`. Potentially this variable is no longer needed at all? And where it was used we can directly invoke the success callback

**Comment by mchv** [comment]
@shtukas I think this is the same with `isUserInContributionsAbTest` and `isUserInContributionsAbTest`, they are not needed anymore. Meaning you may be able to remove  `check-mediator-checks,js` and through it even more code that depends on it.

File: static/src/javascripts/bootstraps/standard/main.js [hunk 6]
```diff
@@ -243,7 +243,6 @@ const bootStandard = (): void => {
         });
     }
 
-    // initilaise the email/outbrain check mediator
     initCheckMediator();
```
Comment (mchv) [inline static/src/javascripts/bootstraps/standard/main.js:246]: @shtukas here for instance `initCheckMediator` may not be needed anymore

**Comment by shtukas** [comment]
Checking that Plista is still alive in Australia against the right build number:



Screenshot 2020-05-04 at 15 20 35

**Comment by shtukas** [comment]
Last Plista check:

Screenshot 2020-05-04 at 16 12 33

Screenshot 2020-05-04 at 16 12 38

**Comment by shtukas** [comment]
And also checking that regular ads are showing :)

Screenshot 2020-05-04 at 16 16 39

File: static/src/javascripts/bootstraps/standard/main.js (hunk 6 above)
Comment (shtukas) [reply static/src/javascripts/bootstraps/standard/main.js]: Done

File: static/src/javascripts/projects/commercial/modules/dfp/define-slot.js (hunk 5 above)
Comment (shtukas) [reply static/src/javascripts/projects/commercial/modules/dfp/define-slot.js:100]: Just seen this, let me check :)

Comment (shtukas) [reply static/src/javascripts/projects/commercial/modules/dfp/define-slot.js:100]: You are right @ioanna0 I will let you do that one in a separate PR :)

**Comment by shtukas** [comment]
Australia PROD check:

Screenshot 2020-05-04 at 16 46 54

File: static/src/javascripts/projects/commercial/modules/dfp/define-slot.js (hunk 5 above)
Comment (shtukas) [reply static/src/javascripts/projects/commercial/modules/dfp/define-slot.js:100]: Done in https://github.com/guardian/frontend/pull/22553

**Pull request merged**